src/main/kotlin/sg/com/quantai/etl/services/CryptoService.kt

```diff
@@ -0,0 +1,120 @@
+package sg.com.quantai.etl.services
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import java.sql.Timestamp
+import java.time.Instant
+
+@Service
+class CryptoService(
```

Comment:
Create one CryptoService to handle the CryptoCurrentyAPI procedures; and let the CryptoService to handle services only related to the transformed data